src/lm_buddy/constants.py

```diff
@@ -0,0 +1,9 @@
+import os
+from pathlib import Path
+
+STORAGE_PATH_ENVIRONMENT_VARIABLE: str = "LM_BUDDY_STORAGE"
```

Comment:
I kind of didn't like using the `RAY_STORAGE` value for the outputs generated by this library, since it's not even guaranteed to be running on Ray. 

Are we happy introducing our own `LM_BUDDY_STORAGE` env var for global control over outputs produced by our functions?